internal/compiler/passes.rs

```diff
@@ -35,8 +35,8 @@ mod lower_shadows;
 mod lower_states;
 mod lower_tabwidget;
 mod lower_text_input_interface;
-mod materialize_fake_properties;
-mod move_declarations;
+pub mod materialize_fake_properties;
+pub mod move_declarations;
```

Comment:
We can't really put comment there because rustfmt will re-order the lines and the comment will just get on the right line when this happen.

You can test this behavior by adding this comment locally,  then add a `mod maa;`   at the end, when you save, the `mod maa;` line will be inserted between the comment and the line it belongs to.